src/main/java/core/basesyntax/impl/ReturnHandlerImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class ReturnHandlerImpl implements OperationHandler {
```

Comment:
There should be no empty line at the beginning of the class definition. Please remove the empty line to comply with the checklist.